lms/templates/sudo/sudo.html

```diff
@@ -0,0 +1,21 @@
+{% extends "main_django.html" %}
+{% load i18n %}
+
+{% block body %}
+<section style="margin: 0 auto; width: 480px; padding: 50px;">
+    <div class="inner-wrapper">
+        <header>
+          <h2 style="text-align: center;">{% trans "Confirm Your Password to Access the Instructor Dashboard" %}</h2>
```

Comment:
I didn't realize that we support standard django templates (usually we use mako). @sarina - can you confirm whether `trans` actually works? I've never seen this before in the platform.